pkgs/development/ocaml-modules/elpi/default.nix

```diff
@@ -5,6 +5,7 @@
 with lib;
 let fetched = import ../../../build-support/coq/meta-fetch/default.nix
   {inherit lib stdenv fetchzip; } ({
+    release."1.13.1".sha256 = "12a9nbdvg9gybpw63lx3nw5wnxfznpraprb0wj3l68v1w43xq044";
```

Comment:
Why isn’t it the default version (the one mentioned on line 3 of this file)?

Reply:
I did not notice there was a default version